R/eda.R

```diff
@@ -116,11 +116,42 @@ describe_num_var <- function(dataframe, num_vars) {
 #' describe_num_var(X, cat_vars)
 #'
 describe_cat_var <- function(dataframe, cat_vars) {
-  #TODO implement function
-  print(cat_vars)
+  # Check the input dataframe is a dataframe
+  if (!is.data.frame(dataframe)) {
+    stop("The value of the argument 'dataframe' should be of type  'data.frame' or 'tibble'.")
```

Comment:
```suggestion
    stop("The value of the argument 'dataframe' should be of type 'data.frame' or 'tibble'.")
```
There is an extra space.